apps/nameguard.io/src/app/docs/displayed-name/page.tsx

```diff
@@ -41,14 +41,14 @@ export default function DisplayedNamePage() {
             <div className="w-full flex justify-center items-center font-mono">
               <pre>
                 {
-                  "<TruncatedText name \n textStylingClasses='text-purple-400' />"
+                  "<TruncatedText name \n textStylingClasses='\nens-webfont font-black' />"
```

Comment:
why the '\n'?

Reply:
For the string to break into a new line

Reply:
@FrancoAguzzi I don't understand why we're doing this here? I've never seen this before in my life it looks very strange.

Reply:
Here I explain to you the usage of `\n` which is called:

![Captura de Tela 2024-06-12 às 09 04 36](https://github.com/namehash/nameguard/assets/49823133/2ca28549-fd4d-4c15-ba07-3b06c30e923f)

Reply:
@FrancoAguzzi I fully understand what '\n' is 😉 What I don't understand is why we're putting explicit newline characters where we are in the code. These seem like a very strange places for explicit newline characters.

Reply:
I get it! The thing is that these are not code lines but these are code documentation lines, so these are plain strings that need to be visually displayed in the screen, thus, they are not code but strings.